perception/autoware_detected_object_validation/src/lanelet_filter/lanelet_filter.cpp

```diff
@@ -100,18 +103,27 @@ void ObjectLaneletFilterNode::objectCallback(
     return;
   }
 
-  // calculate convex hull
-  const auto convex_hull = getConvexHull(transformed_objects);
+  if (transformed_objects.objects.size() > 0) {
```

Comment:
It might be a matter of preference, but this one is easier to read.
```suggestion
  if (!transformed_objects.objects.empty()) {
```

Reply:
fixed https://github.com/autowarefoundation/autoware.universe/pull/8240/commits/6997be6d168cef758f8ca4bb1c0d334ebab661ec